doc/scenario.md

```diff
@@ -29,6 +29,7 @@ In `cartridge_scenario` you should specify names of steps. The default scenario
 There are additional steps that are not included in the default scenario, but can be used in a custom one:
 
 - [set_control_instance](#set_control_instance)
+- [rotate_dists](#rotate_dists)
```

Comment:
What about to include this step in default scenario and make flag to cleanup old packages?

Reply:
I'm not sure, it is used only for TGZ + multiversion like `update_instance`, but `update_instance` is required to start application and for other flow it's just skipped. `rotate_dists` is some optional action that can be performed separately from deploy, and I think that it's convenient way to use it in separated playbook only when you want to remove redundant packages.